frontend/public/components/_resource-dropdown.scss

```diff
@@ -1,6 +1,14 @@
 .co-type-selector {
   margin-right: -2px;
   min-width: 0;
+  .btn-dropdown__content-wrap {
```

Comment:
I'm not sure we need CSS to fix this. Can you try changing the `title` attribute in the dropdown if we want different markup for the selected value?